### PR TITLE
Added gradle-application plugin and renamed top package to io.curity.

### PR DESCRIPTION
**Pull request by renatoathaydes**
Just run this to see the app:

```
./gradlew run
```

Most things don't work yet, but you can have a feel for what's going on so far.

### CODE REVIEW

File: oauthtest-core/build.gradle [hunk 1]
```diff
@@ -7,6 +8,8 @@ repositories {
     jcenter()
 }
 
+mainClassName = "com.athaydes.oauth.core.OAuthTestApplication"
```
Comment (renatoathaydes) [inline oauthtest-core/build.gradle]: I guess I'll have to change the package names.

File: oauthtest-core/build.gradle [hunk 2]
```diff
@@ -7,6 +8,8 @@ repositories {
     jcenter()
 }
 
+mainClassName = "io.curity.oauth.core.OAuthTestApplication"
```
Comment (travisspencer) [inline oauthtest-core/build.gradle]: helpful

**Review by travisspencer — COMMENTED** [review]
The addition of the plugin is helpful. 

Related to the package names, I think the `se.curity` package is a funny joke. Why not use that one if for nothing else then laughs?

File: oauthtest-core/build.gradle (hunk 2 above)
Comment (renatoathaydes) [reply oauthtest-core/build.gradle]: I thought our new package names in Open Source world would be `io.curity`. Will change to `se.curity`, I also like that.

**Pull request merged**